riocli/project/features/dockercache.py

```diff
@@ -52,6 +52,12 @@
     default=(),
     help="Name of the secret for upstream docker registry",
 )
+@click.option(
+    "--data-directory",
+    type=click.STRING,
+    default=(),
+    help="Registry data directory path (default: /opt/rapyuta/volumes/docker-cache/)",
```

Comment:
todo: Specify the value in the default field and remove from the help. The Click library automatically displays the default value.